your-bench.h

```diff
@@ -255,8 +267,8 @@ void exprToYOUR(BenchExpr Expr, map<string,vector<Tensor<double>>> exprOperands,
 			double *data = (double *)mstruct.vals;
 			double *output;
 
-			ATL_TIME_REPEAT(output = (double *) calloc(vdim0, sizeof(double))
-							,  SpMBCSR(data,vvals,crd,pos,cols,rows,blockSize1,blockSize2,output)
+			ATL_TIME_REPEAT(output = (double *) calloc(rows, sizeof(double))
+							,  SpMBCSR(data,xbvals,crd,pos,cols,rows,blockSize1,blockSize2,output)
```

Comment:
I'm not sure if this should be passed cols, rows or rows, cols, but I left it as you have it

Reply:
Yeah it's meant to take cols rows